test/Sentry/SentryLaravelServiceProviderTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Sentry\SentryLaravel;
+
+
+class SentryLaravelServiceProviderTest extends \Orchestra\Testbench\TestCase
+
+{
+
+    protected function getEnvironmentSetUp($app)
+    {
+        // $app['config']->set('services.stripe.secret', getenv('STRIPE_SECRET'));
```

Comment:
Maybe we can set the sentry config with it and make sure the client takes on those configurations?